src/docs/schema.dox

```diff
@@ -362,4 +362,16 @@ Here is another example program, @ex_ref{ex_call_center.c}.
 @snippet ex_call_center.c call-center decl
 @snippet ex_call_center.c call-center work
 
+@section logged_non_logged_tables Logged and Non-logged tables
+
+- WiredTiger writes records to the log for each transaction when logging is enabled
+  using the \c log=(enabled) configuration to \c wiredtiger_open.
+- WiredTiger supports checkpoint durability on non-logged tables and logging
+  is disabled by default. See @ref checkpoint for information on checkpoint durability.
+- WiredTiger supports commit-level durability on logged tables. See @ref durability for
+  information on commit-level durability.
+- Rollback to stable does not operate on logged tables as the updates on these
```

Comment:
How about adding section or something to indicate that the other differences between the logged tables and non-logged tables? and then mentioning the following two points would be good.

Reply:
Added a different section for the last two points.